docs/userguide/collation/api.md

```diff
@@ -238,6 +244,48 @@ delete coll;
 }
 ```
 
+**C++:** (as of ICU 76)
+
+```c++
+ErrorCode status;
+Locale locale = Locale::forLanguageTag("sv", status);
+LocalPointer<Collator> collator(Collator::createInstance(locale, status), status);
+status.assertSuccess(); // Override ErrorCode::handleFailure() to handle failure.
+
+std::vector<std::string> utf8{
+    "Arnold", "Øystein", "Ingrid", "Åke", "Olof", "İsmail", "Örjan",
+};
+
+std::sort(utf8.begin(), utf8.end(), collator->less());
+
+std::vector<UnicodeString> utf16{
+    u"Arnold", u"Øystein", u"Ingrid", u"Åke", u"Olof", u"İsmail", u"Örjan",
+};
+
+std::sort(utf16.begin(), utf16.end(), collator->less());
+```
+
+**C++:** (calling the ICU C API, as of ICU 76)
```

Comment:
please add a line with
```
using namespace U_HEADER_NESTED_NAMESPACE;
```
or U_HEADER_ONLY_NAMESPACE as appropriate

Reply:
The way I had understood the style of the code examples in the userguide was the readers were supposed to figure out on their own what namespace prefixes to add where in order to use these snippets in their own code.

I myself don't think that's a particularly good idea and when now also you doubt that they'll get it right I've now instead updated the code examples added by this PR to instead have the namespace prefixes spelled out so that the snippets can be copy-pasted directly.